CHANGELOG.md

```diff
@@ -0,0 +1,104 @@
+### Changelog
+
+##### Version 2.6.1 (April 13 2018)
+
+-   [JENKINS-19973](https://issues.jenkins-ci.org/browse/JENKINS-19973) Option to hide ssh command in log
+
+##### Version 2.6 (April 7 2018)
+
+-   [JENKINS-46172](https://issues.jenkins-ci.org/browse/JENKINS-46172) Handle NPE when not all fields are specified &
+    save only properly configured sites
+
+-   [JENKINS-10128](https://issues.jenkins-ci.org/browse/JENKINS-10128) Set build to UNSTABLE when no SSH site is
+    configured instead of NPE
+-   Add Jenkinsfile to plugin
+-   bump ssh-credentials dependency to 1.12
+-   Add ajax validation to essential ssh host fields
+
+INFO: the "Add Credentials" button for SSH Site **don't work** in recent
+Jenkins 2.x versions - this will be fixed in upcoming 3.0 version.  
+(major version since plugin will have to migrate it's configuration to
+new format)
```

Comment:
Remove mention of Jenkins 3, since there are no plans for a Jenkins 3 and no work being done on it.

You're welcome to ignore this comment completely, since the goal here was to transfer the changelog, not correct errors in the changelog while transferring it.
```suggestion
Jenkins 2.x versions.
```